packages/xdl/src/Webpack.ts

```diff
@@ -628,8 +628,10 @@ async function loadConfigAsync(
   argv?: string[]
 ): Promise<WebpackConfiguration> {
   setMode(env.mode);
-  // Check if the project has a webpack.config.js in the root.
-  const projectWebpackConfig = path.resolve(env.projectRoot, 'webpack.config.js');
+  // Check if the project has a webpack.config.{ts,js} in the root.
+  const projectWebpackConfig = ["ts", "js"]
+    .map(ext => path.resolve(env.projectRoot, `webpack.config.${ext}`))
+    .find(filePath => fs.existsSync(filePath));
```

Comment:
Maybe just do something like this (a bit more readable and less complicated): 
```ts
const projectWebpackConfig = resolveFrom.silent(env.projectRoot, './webpack.config.ts') || resolveFrom.silent(env.projectRoot, './webpack.config.js');
```

Reply:
Done. Thank you!